hwy/contrib/sort/algo-inl.h

```diff
@@ -525,16 +529,26 @@ void Run(Algo algo, KeyType* HWY_RESTRICT inout, size_t num,
     }
 #endif  // HAVE_VXSORT
 
-    case Algo::kStd:
+    case Algo::kStdSort:
       if (Order().IsAscending()) {
         return std::sort(inout, inout + num, less);
       } else {
         return std::sort(inout, inout + num, greater);
       }
 
+    case Algo::kStdSelect:
+      if (Order().IsAscending()) {
+        return std::nth_element(inout, inout + k, inout + num, less);
```

Comment:
@zbjornson @jan-wassenberg Highway is meant to work with C++11, but std::nth_element was introduced in C++17. Is this okay?

Reply:
Good point, thanks for pointing that out. This is test-only, so I think we could simply wrap all usages in `#if HWY_CXX_LANG >= 201703L`.